etc/transfer-script.sh

```diff
@@ -2,4 +2,10 @@
 # transfer script example
 # /etc/archivematica/automation-tools/transfer-script.sh
 cd /usr/lib/archivematica/automation-tools/
-/usr/share/python/automation-tools/bin/python -m transfers.transfer --user <user> --api-key <apikey>  --ss-user <user> --ss-api-key <apikey> --transfer-source <transfer_source_uuid> --config-file <config_file>
+/usr/share/python/venv/automation-tools/bin/python -m transfers.transfer 
```

Comment:
Hi @jorikvankemenade and @ross-spencer, shouldn't this be `/usr/share/python/automation-tools/venv/bin/python`?

Reply:
You are right @jraddaoui, completely missed that. 

@jorikvankemenade are you able to make the change @jraddaoui requests here and below and submit a new PR on top of the new `master` branch?